src/index.html

```diff
@@ -17,6 +17,90 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="wrapper">
+        <div class="navigation_logo">
+          <a
+            class="logo_link"
+            href="#"
+          >
+            <img
+              class="logo"
+              alt="MOYO company logo"
```

Comment:
The alt attribute of img tag should describe the image content. In your case, it would be better to describe it as 'Logo of MOYO company'.